transport-interop/Makefile

```diff
@@ -1,22 +1,107 @@
+# To build in parallel, run: make all-parallel -j$(nproc) or make parallel -j$(nproc)
+# where $(nproc) is the number of CPU cores available
+# 
+# make all          - builds everything sequentially (original behavior)
+# make all-parallel - builds everything in parallel (including within subdirectories)
+
 GO_SUBDIRS := $(wildcard impl/go/*/.)
 JS_SUBDIRS := $(wildcard impl/js/*/.)
 RUST_SUBDIRS := $(wildcard impl/rust/*/.)
+RUST_CHROMIUM_SUBDIRS := $(wildcard impl/rust-chromium/*/.)
```

Comment:
@achingbrain Not sure why I had to add this

Reply:
Hi Joao @jxs 
I had to add this line other wise there was a complain about missing docker images.
Can you tell why ?
Thanks, Luca